doc/dev/deimos_wavecalib.rst

```diff
@@ -0,0 +1,163 @@
+  detection significance. The FWHM computed in this way will override the assumed **fwhm**, which will still be used
+  as first guess. The default for DEIMOS is **fwhm_fromlines = True**.
+
+See :ref:`pypeit_par:WavelengthSolutionPar Keywords` for all the parameters that guide the wavelength calibration.
+
+Access
+------
+
+Users can access the DEIMOS wavelength calibration information in a couple of ways:
+
+- Inspecting the Quality Assurance (QA) plot generated by ``PypeIt`` and saved in PNG files.
+- Running ``pypeit_chk_wavecalib Masters/MasterWaveCalib_X_X_XX.fits``, which prints wavelength solution
+  information for each slit.
+
+See :ref:`master_wvcalib` for more information.
+
+Testing
+-------
+
+Requirements state:
+
+* PD-20: "As a user, I expect automatic wavelength calibration for all slits to succeed (for narrow slits, 0.7"-2")"
+
+    * PD-21: "Use arcs to provide wavelength solutions for narrow slits (0.7" - 2")"
+
+    * PD-22: "Use special blue arcs to cover the bluest observations"
+
+* PD-37:  "The pipeline produces a wavelength solution as accurate as the one derived starting from an optical model"
+
+
+``PypeIt`` meets these requirements as demonstrated by the tests performed by the ``PyPeIt`` team and described below.
+
+Description
+^^^^^^^^^^^
+
+We performed the wavelength calibration on a large sample of 36 DEIMOS datasets spanning a wide range of wavelengths,
+gratings, central wavelengths, and slit widths:
+
+======== ==========================================================================================================
+Grating    Central wavelength (slit width)
+======== ==========================================================================================================
+600ZD     5500 |AA| (1"), 6000 |AA| (1.5"), 6100 |AA| (1"), 6300 |AA| (1"), 6500 |AA| (1"), 7000 |AA| (0.8"),
+          7200 |AA| (1"), 7500 |AA| (1"), 7900 |AA| (1"), 8000 |AA| (1.2")
+830G      6720 |AA| (1"), 7500 |AA| (1"), 7800 |AA| (1"), 8100 |AA| (0.6"), 8100 |AA| (1"), 8580 |AA| (1"),
+          8580 |AA| (1"), 9000 |AA| (1")
+900ZD     5500 |AA| (0.7"), 6000 |AA| (1"), 8000 |AA| (0.75")
+1200G     5500 |AA| (1"), 6700 |AA| (1"), 7200 |AA| (1"), 7750 |AA| (0.7"), 8800 |AA| (1")
+1200B     5200 |AA| (1"), 5280 |AA| (1"), 5500 |AA| (0.85")
+======== ==========================================================================================================
+
+Including six datasets using LVM slitmask, which is a special slitmask composed of 5 slits with different widths
+(0.7", 0.8", 1.", 1.2", 1.5"). These datasets were taken with the following gratings (and central wavelengths):
+830G (8400 |AA|), 600ZD (7100 |AA|), 830G (7900 |AA|), 1200G (7800 |AA|), 1200B (5450 |AA|), 1200B (5200 |AA|).
+
+Results
+^^^^^^^
+
+Out of the **4072 slits** that have been tested, for only **37 (0.9%)** the wavelength calibration failed or
```

Comment:
Maybe point out that these can be fixed using the `identify` GUI.  Or I hope they can. :)

Reply:
Done.